### PR TITLE
update ibc-go to v8

### PR DESCRIPTION
**Pull request by dongrie**
Made a series of modifications to support ibc-go/v8

### CODE REVIEW

File: modules/light-clients/xx-mock/types/client_state.go [hunk 1]
```diff
@@ -40,21 +42,22 @@ func (cs ClientState) GetLatestHeight() exported.Height {
...
-	clientStore sdk.KVStore,
+	clientStore storetypes.KVStore,
 	cdc codec.BinaryCodec,
 	height exported.Height,
 ) (uint64, error) {
 	// get consensus state at height from clientStore to check for expiry
 	consState, found := getConsensusState(clientStore, cdc, height)
 	if !found {
-		return 0, sdkerrors.Wrapf(clienttypes.ErrConsensusStateNotFound, "height (%s)", height)
+		//return 0, sdkerrors.Wrapf(clienttypes.ErrConsensusStateNotFound, "height (%s)", height)
```
Comment (siburu) [inline modules/light-clients/xx-mock/types/client_state.go]: Is there reason why this comment is left?

**Review by siburu — COMMENTED** [review]
@dongrie Almost OK, but I made a question.

File: modules/light-clients/xx-mock/types/client_state.go (hunk 1 above)
Comment (dongrie) [reply modules/light-clients/xx-mock/types/client_state.go]: https://github.com/datachainlab/ibc-mock-client/pull/13/commits/523df2202b5f4cc18db1d4048f4c5bfba4888817
Removed comment

**Review by siburu — APPROVED** [review]
LGTM. Thank you.

**Pull request merged**